src/components/Metrics.js

```diff
@@ -15,7 +22,25 @@ const Metrics = React.memo(function Metrics({
 }) {
   const { layoutName } = useLayout()
   const compactMode = layoutName === 'small'
-  const { requestToken, stakeToken } = useAppState()
+  const { requestToken, stakeToken, currencies } = useAppState()
```

Comment:
It may make more sense to use the currency hook directly from this component as opposed to having it in the app state context since we will be using it only when the Metrics component is rendered. If in the future we see its useful to have as global state we could consider lifting the state up.